tests/phpunit/admin/class-wc-rest-stripe-orders-controller-test.php

```
@@ -252,6 +252,66 @@ public function test_capture_payment_stripe_error() {
+		$request  = new WP_REST_Request( 'POST', $endpoint );
+		$request->set_param( 'payment_intent_id', 'pi_12345' );
+		$response = rest_do_request( $request );
+
+		$this->assertEquals( 200, $response->get_status() );
+
+		$stored = WC_Stripe_Order_Helper::get_instance()->get_stripe_ipp_channel( $order );
+		if ( '' === $expected ) {
+			$this->assertEmpty( $stored );
+		} else {
+			$this->assertEquals( $expected, $stored );
+		}
```

PHPMD (2.15.0)

[error] The method test_capture_payment_ipp_channel_storage uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)